src/scripts/main.js

```diff
@@ -2,10 +2,21 @@
 
 function waitFor(element, eventName) {
   // write your code here
+  return new Promise((resolve, reject) => {
```

Comment:
You don't use 'reject' in your code, so you can remove it from the function argument because we don't need it.